src/components/feature.mjs

```diff
@@ -0,0 +1,83 @@
+import { parseExample }  from '../../utils/feature-example.js';
+import { backTicksToCodeTags, escapeHTML, html } from '../util/html.mjs';
+import { renderCaniuseFeature } from './caniuse-feature.mjs';
+import { renderFeaturePolyfills } from './feature-polyfills.mjs';
+
+const stages = [
+	'stage-0-aspirational',
+	'stage-1-experimental',
+	'stage-2-allowable',
+	'stage-3-embraced',
+	'stage-4-standardized',
+]
```

Comment:
Super minor (might be elsewhere) but this is lacking semi colon which is not a big deal but just flagging for consistency, perhaps it should be enforced?

Reply:
Good catch!

Unsure if we even have a linter setup here 🤔